src/server/api/controllers/studentsprofile.controller.js

```diff
@@ -0,0 +1,17 @@
+const knex = require('../../config/db');
+
+const getStudentsProfile = async () => {
+  try {
+    const studentsProfile = await knex('users')
+      .select('name', 'group_id', 'groups.name as group_name', 'timezone')
+      .leftJoin('groups', 'group_id', 'groups.id')
+      .leftJoin('groups', 'group_name', 'groups.name');
+    return studentsProfile;
```

Comment:
Is this returning just one profile i.e. for the logged in user? Or is it returning a list of profiles for all users?

Reply:
nothing returns. only '404 Sorry can't find that!' shows on Response body

Reply:
Well, you need to test it with Postman with instructions from #255.